include/convex_bodies/spectrahedra/spectrahedron.h

```diff
@@ -250,6 +254,14 @@ class Spectrahedron {
         return std::pair<NT, int> (pos_inter, -1);
     }
 
+    std::pair<NT, int> line_positive_intersect(PointType const& r,
```

Comment:
which style are you using here?
```
function {
...
}
```
or
```
function
{
...
}
```

Reply:
I see that the file also uses two styles and this is confusing. Thus you can use whatever style you want here.